src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -121,6 +121,21 @@
         GROUP BY genetic_profile.stable_id, genetic_profile.name, sample_mv.cancer_study_identifier;
     </select>
     
+
+    <!-- for /sample-lists-counts/fetch (returns CaseListDataCount) -->
+    <select id="getCaseListDataCounts" resultType="org.cbioportal.model.CaseListDataCount">
```

Comment:
Have you profiled this query?

Does it make sense to use the sample_list_columnstore?

Reply:
interesting.  i wonder who made sample_list_columnstore?    it's already subsecond as is but might as well use this.